2022-07_sprint/schedule_table.md

```diff
@@ -0,0 +1,10 @@
+
+| Event                                     | Date               | Time                    | Duration |  Registration form  |
+|-------------------------------------------|--------------------|-------------------------|----------|---------------------|
+| Open Source Pre-series Office Hours       | Jul 2 (Saturday)   |  9 ET / 13 UTC / 16 EAT | 1 hour   |
+| Open Source Working Session #1            | Jul 9 (Saturday)   |  9 ET / 13 UTC / 16 EAT | 3 hours  |
+| Open Source Working Session #2            | Jul 22 (Friday)    | 12 ET / 16 UTC / 19 EAT | 3 hours  |
+| Open Source Working Session #3 [^1]       | Aug 4/5 (Thur/Fri) | 19 ET / 23 UTC          | 3 hours  |
+| Open Source Post-series Office Hours [^1] | Aug 18 (Saturday)  | 19 ET / 23 UTC          | 1 hour   |
```

Comment:
yep, I would put in only on the schedule page only. This schedule_table page has only the table so we can the include it in both the homepage and the schedule page without duplicating the content.

Reply:
schedule page is fine.  As long as it's it's some place

Reply:
I embedded this inside a dropdown in https://pymc-data-umbrella--54.org.readthedocs.build/en/54/2022-07_sprint/schedule.html, but the embedded square is super small (independently of the embedding being inside a dropdown or not). Do you know how to make it a bit larger? If it is so small it might be better to add a link  directly

Reply:
@OriolAbril For adjusting, we can add a height / width option. 
Here is an example:
```html
<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://reshamas.github.io/assets/images/covid/annot_aut_cze_daily_case_count.html" height="500" width="1000">
</iframe>	
```